tests/test_lightningd.py

```diff
@@ -2727,9 +2731,10 @@ def test_withdraw(self):
         # Make sure bitcoind received the withdrawal
         unspent = l1.bitcoin.rpc.listunspent(0)
```

Comment:
We could do `l3.rpc.listoutputs` instead, and change the assertions as needed.

However we want to check using bitcoind later anyway, so this is OK.